code/Controllers/ContentController.php

```diff
@@ -289,14 +289,6 @@ public function getMenu($level = 1)
         return new ArrayList($visible);
     }
 
-    /**
-     * @return ArrayList<SiteTree>
-     */
-    public function Menu($level)
-    {
-        return $this->getMenu($level);
-    }
```

Comment:
This method only exists to work around the fact that `$Menu(2)` doesn't pass the `2` arg to `getMenu()`.
I've changed it so that the args _are_ passed into getters, so this method isn't needed anymore.